views/sms/index.js

```diff
@@ -241,7 +241,7 @@ exports.respond = function(req, res) {
             return save_utils.countRequestsPerRegion(region_code);
         }).then( function(numRequests) {
             requestData = save_utils.createSerial(numRequests, req.cookies.request_object, region_code);
-            requestData.is_sms = 'sms';
+            requestData.origin = 'sms';
```

Comment:
Oh, wait, I guess I was wrong?  This is the change where we are re-naming the request field from "is_sms" to "origin"?  In that case, let's use "origin" for both sides, or "source" for both sides.